Gemfile.lock

```diff
@@ -408,10 +408,10 @@ GEM
       rubocop (~> 1.41)
     rubocop-factory_bot (2.25.1)
       rubocop (~> 1.41)
-    rubocop-performance (1.20.2)
+    rubocop-performance (1.21.0)
       rubocop (>= 1.48.1, < 2.0)
-      rubocop-ast (>= 1.30.0, < 2.0)
     rubocop-rails (2.24.1)
+      rubocop-ast (>= 1.31.1, < 2.0)
```

Comment:
I think this may have moved due to a resolved merge conflict. This `rubocop-ast` dependency belongs to `rubocop-performance`.